conbench/api/compare.py

```diff
@@ -275,7 +339,9 @@ def get(self, compare_shas):
         try:
             baseline_sha, contender_sha = compare_shas.split("...", 1)
         except ValueError:
+            # Note(JP): this cannot raise ValueError.
```

Comment:
This is a TODO. I touched so many things like that today, but there was a line to draw :).